### PR TITLE
Inno setup

### PR DESCRIPTION
**Pull request by jmgrady**
Create script to build `git-crypt` for Windows 10 and build an installer package for it.

### CODE REVIEW

**Comment by jasonleenaylor** [comment]
I'd rather not include the setup.exe under source control, but instead create a release in the repo. You should be able to add a link to the latest release in the readme .
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository

File: git-crypt-setup.iss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Compression=lzma
+SolidCompression=yes
+WizardStyle=modern
+
+[Languages]
+Name: "english"; MessagesFile: "compiler:Default.isl"
+
+[Files]
+Source: "git-crypt\{#MyAppExeName}"; DestDir: "{app}"; Flags: ignoreversion
+Source: "C:\dev\msys64\mingw64\bin\libcrypto-1_1-x64.dll"; DestDir: "{app}"; Flags: ignoreversion
```
Comment (jasonleenaylor) [inline git-crypt-setup.iss]: I'm not sure about these source locations, is dev the default install location for msys64?

Comment (jmgrady) [reply git-crypt-setup.iss]: C:\dev\msys64 is the location included in the instructions I referenced for installing `g++` (https://github.com/orlp/dev-on-windows/wiki/Installing-GCC--&-MSYS2).  The instructions state:
> Make sure to set the install directory to C:\dev\msys64

I could change the setup script to prompt for the location of the msys64 directory instead.

**Comment by jmgrady** [comment]
> I'd rather not include the setup.exe under source control, but instead create a release in the repo. You should be able to add a link to the latest release in the readme .
> https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository

I will look into this.  I agree, I would rather not have the setup.exe under source control - the link to download it is a little cumbersom.

File: git-crypt-setup.iss (hunk 1 above)
Comment (jasonleenaylor) [reply git-crypt-setup.iss]: No, it is fine. I didn't dig into those instructions.

**Review by jasonleenaylor — APPROVED** [review]
:LGTM:

**Pull request merged**